bigbluebutton-html5/imports/ui/components/pads/service.js

```diff
@@ -85,6 +89,60 @@ const getPadContent = (externalId) => {
   return '';
 };
 
+async function convertAndUpload(externalId) {
+  const padId = await makeCall('getPadId', externalId);
+
+  // Build export URL in the desired format
+  const params = getParams();
+  
+  // This returns a blank PDF
+  const exportUrl = Auth.authenticateURL(`${PADS_CONFIG.url}/p/${padId}/export/pdf?${params}`);
+
+  // This returns 403 forbidden
+  // const exportUrl = `${PADS_CONFIG.notes_endpoint}/${padId}/export/pdf`;
+
+  console.log(exportUrl);
+  console.log(`${PADS_CONFIG.notes_endpoint}/${padId}/export/pdf`)
+
+  let exportedSharedNotes = await axios({
+    method: 'get',
+    url: exportUrl,
+    responseType: 'json',
+  });
```

Comment:
Looks like you are trying an outside approach, using the user current session to export the note instead of the internal proxy.

You will have to include the user's Etherpad sessionID cookie at this call.

Reply:
And I'm not so sure about the JSON response type either

Reply:
A final suggestion would be to push all this business logic and custom UI to a new sub-folder within the `notes` folder, with it's own container/component/styles and service files.

Reply:
And I believe only the presenter is capable of uploading files in BBB. I think you might be testing for moderator

Reply:
Aaaaaand if you are really going for the remote download you can use the browser's fetch method instead of importing `axios`.

`axios` still exists as a dependency in BBB because the Node version we use do not support `fetch`

Reply:
Thanks you for the review!
I'll apply those changes. I would prefer doing it through the internal proxy, though, do you happen to know why `http://127.0.0.1:9002/p/${pad}/export/pdf` yields 403?

Reply:
Maybe an incorrect padId